File: Guru99MavenFW/src/test/java/com/guru99/pageObjects/LoginPage.java
package com.guru99.pageObjects;

import java.util.concurrent.TimeUnit;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.PageFactory;

public class LoginPage {
	
	
	  WebDriver ldriver;
	  
	  public LoginPage(WebDriver rdriver) {
	  ldriver = rdriver;
	  PageFactory.initElements(rdriver, this); 
	  }
	 
	//Xpaths  
	public static String UserName_Xpath = "//input[@name='uid']";
	public static String Password_Xpath = "//input[@name='password']";
	public static String btnLogin_Xpath = "//input[@name='btnLogin']";
	
	
	public void setUserName(String uname) {
		ldriver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
		ldriver.findElement(By.xpath(UserName_Xpath)).clear();
		ldriver.findElement(By.xpath(UserName_Xpath)).sendKeys(uname);
	}	
	
	public void setPassword(String pwd) {
		ldriver.findElement(By.xpath(Password_Xpath)).clear();
		ldriver.findElement(By.xpath(Password_Xpath)).sendKeys(pwd);
	}
	
	public void clickSubmit() {
		ldriver.findElement(By.xpath(btnLogin_Xpath)).click();
	}
}

File: PythonPycharm/Python/FirstProject/ReadDataJson/pageobjectmodel/LoginPage.py
from selenium.webdriver.common.by import By


class LoginPage:
    
    username_testbox_id = "username"
    password_txtbox_id = "password"
    login_button_xpath  = "//button[@type='submit']"
    
    def __init__(self, driver):
        self.driver = driver
        
    def enter_username(self, userName):
        self.driver.find_element(By.NAME, self.username_testbox_id).clear()
        self.driver.find_element(By.NAME, self.username_testbox_id).send_keys(userName)
            
    def enter_password(self, password):
        self.driver.find_element(By.NAME, self.password_txtbox_id).clear()
        self.driver.find_element(By.NAME, self.password_txtbox_id).send_keys(password)
        
    def click_login_button(self):
        self.driver.find_element(By.ID, self.login_button_xpath).click()